### DISCUSSION TITLE
同步程式

### DISCUSSION THREAD

**Discussion by 111426027** [1101-GS4719-C/Python-Community-GS4719-C#1700] [General]
請問有人記得要如何將github上的作業同步到VS code嗎?(之前上課忘記錄影了，上網查都是舊的方法)

**Comment by ATREE01** [comment]
確認你的電腦有安裝git之後在vscode有一個source controle的地方點下去後會clone repository選擇clone from github這邊應該會要求你先登入你的github帳號
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/100257906/d4d164fb-7aed-4820-9e47-873cffe8a654)
接下來在選擇你想要的Repository就好了
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/100257906/22ec74a2-934f-426d-a478-f25962a19192)

**Comment by 12Jesse24** [comment]
印象中老師有把操作影片放在eeclass上，如果卡住的話可以去看看，應該是準備開發環境中的第三個影片。
[機房小教室｜Git & Github Hands-on](https://ncueeclass.ncu.edu.tw/media/doc/126330)

**Comment by mrweixd** [comment]
如果你的系統是mac的話，理論上會卡在驗證的步驟。中文版的參考解決方法在這：https://blog.csdn.net/qq_32614525/article/details/121124216
我被這個卡蠻久的。
